src/DonationForms/resources/propTypes.ts

```diff
@@ -40,6 +40,10 @@ export interface CheckboxProps extends FieldProps {
     value: string | number;
 }
 
+export interface TextareaProps extends FieldProps {
+    helpText?: string;
+}
```

Comment:
I don't think we should refer to this as "help text", it should just be a description, which I believe we already have an interface for.

Reply:
Oh, I see the issue. We don't have a "description" concern in the Fields API. Crud. And the Help Text is being used all over the place.

I'm inclined to say let's move this along here and revisit it after Next Gen is merged into core so we don't keep going back and forth. Sound good?

Reply:
yeah helpText already exists, we can additionally add description when that's ready in core & just use that in the template.  I think it's fine for now though